pkg/analytics/analyticstracker.go

```diff
@@ -0,0 +1,70 @@
+package analytics
+
+import (
+	"time"
+
+	"github.com/okteto/okteto/pkg/model"
+)
+
+const deployEvent = "Deploy"
+
+type AnalyticsTracker struct {
+	TrackFn func(event string, success bool, props map[string]interface{})
+}
+
+func NewAnalyticsTracker() *AnalyticsTracker {
+	return &AnalyticsTracker{
+		TrackFn: track,
+	}
+}
+
+// DeployMetadata contains the metadata of a deploy event
+type DeployMetadata struct {
```

Comment:
can we separate the deploy/destroy to a `track_deploy` `track_destroy` file and leave here just the Analytics Tracker? i can use this for the Up events....

Reply:
Addressed. Let me know what you think about the change